### PR TITLE
Added rating component

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Please review my rating component
PREVIEW : https://vercel-ui-preview.netlify.app/component-lib/rating.html

### CODE REVIEW

File: component-lib/rating.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+	<body>
+		<main class="main-container">
+			<header class="header">
+				<div class="flex-hz">
+					<button class="btn btn-icon btn-menu" onclick="openMobileSidebar()">
+						<i class="fa fa-bars fa-lg" aria-hidden="true"></i>
+					</button>
+					<a class="flex-hz link-no-style" href="../index.html">
+						<div class="logo-container">
+							<img class="img-round" src="./logo.png" />
```
Comment (Naman-Saxena1) [inline component-lib/rating.html]: You should add alt="" on your img tags

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: component-lib/rating.html (hunk 1 above)
Comment (nikhiljugale007) [reply component-lib/rating.html]: Added alt for images

File: component-lib/rating.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+				<div class="flex-hz">
+					<a href="./installation.html" class="link-no-style">
+						<button
+							class="btn btn-outlined"
+							onclick="location.href='documentation.html'"
+						>
+							Documentation
+						</button>
+					</a>
+					<button class="btn btn-icon" onclick="changeTheme()">
```
Comment (mehulsatardekar) [inline component-lib/rating.html:71]: avoid adding JS here, make Js file for that.

**Review by mehulsatardekar — CHANGES_REQUESTED** [review]

**Comment by mehulsatardekar** [comment]
rest the code is good 👌

File: component-lib/rating.html (hunk 2 above)
Comment (nikhiljugale007) [reply component-lib/rating.html:71]: I cannot change my folder structure now. So I am planning to change the folder structure as I complete all components.
I will keep this in mind.

File: component-lib/rating.html [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Rating | Docs</title>
+		<link rel="stylesheet" href="./all-comp.css" />
+		<link
+			href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"
+			rel="stylesheet"
+		/>
+		<script>
+			var themeDefault = window.localStorage.getItem("data-theme");
```
Comment (kaustubh-jsr) [inline component-lib/rating.html]: You can use "let" or "const" here, instead of var. Rest looks good to me.

Comment (nikhiljugale007) [reply component-lib/rating.html]: Made the changes

**Pull request merged**